addons/sys_zeus/fnc_handleZeusSpeakPressUp.sqf

```diff
@@ -15,12 +15,13 @@
  * Public: No
  */
 
-[{!(GVAR(keyDownWait))}, {
-    player setVariable [QGVAR(inZeus), false, true];
+if (!canSuspend) exitWith {0 spawn FUNC(handleZeusSpeakPressUp)};
 
-    // Restore previous out of Zeus state
-    [player getVariable [QGVAR(wasSpectator), ACRE_IS_SPECTATOR]] call EFUNC(api,setSpectator);
+sleep 0.5;
```

Comment:
No.

Why is this necessary? Does it require just a frame delay? Does it require a specific delay? What is that dependent on?

Reply:
I don't like this either, but every other solution I tried didn't work.

Basically, if the key is pressed really quickly, it would start transmitting but never stop. I am very much open to a better solution, I just couldn't get any others to work consistently.